Toivoa_Frontend/src/components/loginRegistryPage/Create.js

```diff
@@ -1,214 +1,173 @@
+            <li>
+              <label>Username:</label>
+              <input
+                {...usernameField}
+                placeholder="Enter your account username"
+              />
+            </li>
+            <li>
+              <label>Email:</label>
+              <input {...emailField} placeholder="Enter your email" />
+            </li>
+            <li>
+              <label>Phone Number:</label>
+              <input
+                {...phoneNumberField}
+                placeholder="Enter your phone number"
+              />
+            </li>
+            <li>
+              <label>First Name:</label>
+              <input {...firstNameField} placeholder="Enter your first name" />
+            </li>
+            <li>
+              <label>Last Name:</label>
+              <input {...lastNameField} placeholder="Enter your last name" />
+            </li>
+            <li>
+              <label>Password:</label>
+              <input {...passwordField} placeholder="Enter your password" />
+            </li>
+            <li>
+              <label>Account Type:</label>
+              <select {...accountTypeField}>
+                <option defaultValue>Select account type</option>
+                <option value="consumer">Consumer</option>
+                <option value="seller">Seller</option>
+                <option value="market vendor">Marketer Vendor</option>
+              </select>
+            </li>
+            <li>
+              <label>Country Code:</label>
+              <select {...countryCodeField}>
+                <option defaultValue>Select country</option>
+                {countryCodes.map((country) => (
+                  <option key={country.code} value={country.code}>
+                    {country.name}
+                  </option>
+                ))}
+              </select>
+            </li>
+            <li>
+              <label>Location:</label>
+              <input {...locationField} placeholder="Enter your location" />
+            </li>
+            <li>
+              <label>Age:</label>
+              <input {...ageField} placeholder="Enter your age" type="number" />
+            </li>
+            <li>
+              <label>Gender:</label>
+              <select {...genderField}>
+                <option defaultValue>Select gender</option>
+                <option value="Male">Male</option>
+                <option value="Female">Female</option>
+                <option value="Other">Other</option>
+              </select>
+            </li>
+            <li>
+              <button type="submit">Submit</button>
+            </li>
+            <li>
+              <a onClick={switchToLogin} className="switch-to-create">
+                Switch to login
+              </a>
+            </li>
+          </ul>
+        </form>
+      </div>
+      <button onClick={closeEvent} className="closeButton">
+        <h1>x</h1>
+      </button>
+    </div>
+  );
 };
 
 export default CreateAccount;
```

Comment:
Please use the latest version of the createField. You can find them here.
https://github.com/Minimal-Alexi/ProjectWB/tree/sprint-3/Toivoa_Frontend/src/components/loginRegistryPage